src/app/project/board/address-template/address-template.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-address-template',
+  template: `
+      <div class="sub-details">
+          <h4>Address:</h4>
+          <address class="contents">
+              <p *ngIf="address['streetAddress']">{{ address['streetAddress'] }}</p>
```

Comment:
I also prefer to use dot notation as you said it is easier.
But in this case if I use it, it gives me variable undefined error. I am not sure why... so I started to use bracket notation and it solved the error.

Reply:
Is this happening everywhere? This is strange, because you use hardcoded data. However you can also use `?` operator to check if key is undefined, then it shouldn't return error if it is:

`address.streetAddress?.anotherProp?`

But if you use primitive object type and get this error: `Identifier 'test' is not defined. 'object' does not contain such a member` then it is clear as error says. So you can leave it now, but in second iteration when you add interfaces, bracket notation should be easily replaced with dot notation.